backends/generators/sverilog/sverilog_generator.py

```
@@ -0,0 +1,179 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+import logging
+from pathlib import Path
+
+# Add parent directory to path for imports
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from generator import load_instructions, load_csrs
+
+
+def format_instruction_name(name):
+    """Format instruction name for SystemVerilog (uppercase with underscores)."""
+    # Handle compressed instructions
+    if name.startswith("c."):
+        name = "C_" + name[2:]
+    # Replace dots with underscores and convert to uppercase
+    return name.replace(".", "_").upper()
+
+
+def format_csr_name(name):
+    """Format CSR name for SystemVerilog (uppercase with underscores)."""
+    return "CSR_" + name.replace(".", "_").upper()
+
+
+def match_to_sverilog_bits(match_str, is_compressed=False):
+    """Convert a match string to SystemVerilog bit pattern."""
+    if not match_str:
+        return "32'b" + "?" * 32
```

Shouldn't this throw an error/warning instead of silently generating a match string of all wildcards that will match any instruction? That would be very problematic in use because it might shadow an actual match later.